eng/common/pipelines/templates/steps/daily-dev-build-variable.yml

```diff
@@ -22,3 +22,7 @@ steps:
     echo "##vso[task.setvariable variable=SetDevVersion]$setDailyDevBuild"
   displayName: "Setup Versioning Properties"
   condition: eq(variables['SetDevVersion'], '')
+- pwsh: |
+    echo "##vso[task.setvariable variable=SetDevVersion;isOutput=true]$(SetDevVersion)"
```

Comment:
Why not do both variable setting in the same step?

Reply:
Because the ```name:``` property gets prepended to the variable name. So I'm doing this to keep existing usages.

Reply:
Interesting the name get prepended even for variables that aren't marked as output? I would have never expected that.

Reply:
Yeah ... that is why it caught me out.